Makefile.inc

```diff
@@ -4,17 +4,18 @@ deploytool ?= operator
 registry_inmemory ?= true
 release_tag ?= latest
 repo ?= quay.io/submariner
+lighthouse ?= fasle
```

Comment:
As discussed in today’s call, we shouldn’t add a variable here; downstreams should use `CLUSTERS_ARGS` etc. to set the Lighthouse options.

Reply:
Done.